var/spack/repos/builtin/packages/py-cgal-pybind/package.py

```diff
@@ -22,3 +23,7 @@ class PyCgalPybind(PythonPackage):
     depends_on("eigen")
     depends_on("py-pybind11")
     depends_on("py-numpy@1.12:", type=("build", "run"))
+    depends_on('py-pytest', type='test')
+
+    def test_install(self):
```

Comment:
```suggestion
    @run_after('install')
    @on_package_attributes(run_tests=True)
    def test_install(self):
```
Without these decorators, that method is not going to get called ever.